test/test-StatsLogger.js

```diff
@@ -410,5 +410,43 @@ describe('StatsLogger', function() {
     }, 1050);
   });
 
+  it('should be able to add a captureFunction', function(done) {
+    var a = statsLogger.createInstance(60, backend, {}, {});
+    a.addStat("myStat", "counter", {initialValue: 0});
```

Comment:
no linter setup in this project.  i added eslint but came up with 1746 errors, so I'm going hold off on that.